magento/adobe-stock-integration
# Introduce massaction mode for media gallery

### Description (*)


**whats left:** 
1. Send request to deleting images :heavy_check_mark: 
2. Show count of selected images :heavy_check_mark: 
3. Correct styles to match mockups :heavy_check_mark:
4. Switch to mass action mode by checkboxes :heavy_check_mark: 
5. On press cancel button terminate massaction mode :heavy_check_mark: 
### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration#1488: Introduce ability to switch to massaction mode in the media gallery
2. Fixes magento/adobe-stock-integration#1516: Add ability to select images clicking on the image tile in the massaction mode
3. Fixes magento/adobe-stock-integration#1513: Display content usage information for multiple assets during multiple delete
### Manual testing scenarios (*)
1. Open Media Gallery page or slide panel
2. Click on massaction checkbox verify mass action mode activated
2. Select images in different folder or pages -> assert selected count up to date with selected images.
3. Click delete selected button 
4. Assert images deleted, assert massaction mode not active anymore
![Screenshot_2020-06-19 Manage Gallery Media Content Magento Admin(2)](https://user-images.githubusercontent.com/2481206/85152814-1e1b5880-b25e-11ea-9049-106af12de295.png)
